en/docs/updates/faq.md

```diff
@@ -26,7 +26,15 @@ In production environments, WSO2 will announce urgent security fixes to customer
 2. Next, transfer the created zip file securely to your isolated environment(s).<br>
 3. Then run the **apply-update** command pointing the proper zip file location. If you have an older update client tool version, at this point the tool will show a message to re-run the same command.<br>
 [The first run of `apply-update` command will self-update the tool]
-4. Upon running the `apply-update` command again will propagate new updates to that environment easily.    
+4. Upon running the `apply-update` command again will propagate new updates to that environment easily.
+
+##### What should you do if you get an`unknown command "apply-update" for "wso2update` error while applying updates offline?
+The reason this error is displayed is because the update tool used is of an older version and is not supported by the new Updates tool. To resolve this problem keenly follow the steps given below:<br>
```

Comment:
Shall we change the first sentence to " The reason this error is displayed is because the update tool used is of an older version and does not support the apply-update command."